tf_files/configs/kube-setup-cronjobs.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# Initializes the Gen3 k8s cronjobs.
+#
+
+set -e
+
+_KUBE_SETUP_CRONJOBS=$(dirname "${BASH_SOURCE:-$0}")  # $0 supports zsh
+# Jenkins friendly
+export WORKSPACE="${WORKSPACE:-$HOME}"
+export GEN3_HOME="${GEN3_HOME:-$(cd "${_KUBE_SETUP_CRONJOBS}/../.." && pwd)}"
+
+if ! kubectl get cronjob google-manage-keys > /dev/null 2>&1; then
+   kubectl create -f "${GEN3_HOME}/kube/services/jobs/google-manage-keys-cronjob.yaml"
```

Comment:
Please rebase on master, and switch to using `g3kubectl` throughout.  The latest code uses `g3kubectl` - which is a wrapper that switches namespaces based on the KUBECTL_NAMESPACE environment.  That setup makes it easier for us to run these scripts in a Jenkins pipeline ...

Reply:
gotcha, will do